Cargo.toml

```diff
@@ -22,21 +22,38 @@ arrow = { version = "*", default-features = false, optional = true}
 default = ["half"] # TODO: remove this as default feature as soon as https://github.com/CodSpeedHQ/codspeed-rust/issues/1 is fixed
 
 [dev-dependencies]
+# rstest = { version = "0.16", default-features = false}
+# rstest_reuse = "0.5"
```

Comment:
Is  something I experimented with and will use in a future PR (parameterizing the tests)